### PR TITLE
Autox

### PR DESCRIPTION
**Pull request by Alan-Cha**

### CODE REVIEW

File: autox/service.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package autox
```
Comment (sriumcp) [inline autox/service.go]: Rename `service.go` to `watcher.go`. 

Remove sub folder `watcher`. Put everything under the `autox` package.

File: autox/service.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+// service.go - entry point for A/B(/n) service; starts controller watching resources
```
Comment (sriumcp) [inline autox/service.go]: Fix.

Comment (sriumcp) [reply autox/service.go]: Also add `doc.go` for package level documentation.

Comment (sriumcp) [reply autox/service.go]: FYI. https://go.dev/blog/godoc

File: cmd/autox.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import (
+	ia "github.com/iter8-tools/iter8/action"
+	"github.com/iter8-tools/iter8/driver"
+
+	"github.com/spf13/cobra"
+)
+
+// autoxDesc is the description of autox cmd
+const autoxDesc = `
+Run the Iter8 AutoX service.
```
Comment (sriumcp) [inline cmd/autox.go]: `autox controller` everywhere instead of `autox service`.

File: action/autox.go [hunk 4]
```diff
@@ -0,0 +1,25 @@
+package action
```
Comment (sriumcp) [inline action/autox.go]: No need for this file.

File: cmd/autox.go [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+// newAutoXCmd creates the autox command
+func newAutoXCmd(kd *driver.KubeDriver) *cobra.Command {
+	actor := ia.NewAutoXOpts(kd)
+
+	cmd := &cobra.Command{
+		Use:   "autox",
+		Short: "Start the Iter8 AutoX service",
+		Long:  autoxDesc,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			return actor.LocalRun()
```
Comment (sriumcp) [inline cmd/autox.go]: Call `autox.Start` here (instead of actor.LocalRun).

**Review by sriumcp — CHANGES_REQUESTED** [review]
Please see change requests.

**Comment by Alan-Cha** [comment]
See #1328

**Pull request closed without merging**